### ISSUE TITLE
A question about the data split !

### ISSUE THREAD

**Issue by caoxu915683474** [jiesutd/NCRFpp#99]
Thanks for sharing this projection! 
I have a question about the data you used to train the model. 

train the model use training set combine with dev set and the result is on test set? 
or train the model only on the training set?
because I find that on this web [page,](https://github.com/sebastianruder/NLP-progress/blob/master/english/named_entity_recognition.md) the reference said you use the combination of training and dev set. And many papers also use the combination of the data set.
I get confused ! can you help me?

**Comment by jiesutd** [comment]
I only used the train dataset to train the model. And the link does not show that my model use the combination, you may see the page carefully.

**Comment by caoxu915683474** [comment]
Oh! I see the ♦ models use the combination, sorry....
